packages/intrepid2/src/Cell/Intrepid2_CellTools.hpp

```diff
@@ -102,8 +102,9 @@ namespace Intrepid2 {
   */
 
 
-  template<typename ExecSpaceType>
+  template<typename DeviceType>
```

Comment:
Let's not change this if we do not fix the core part. This ``CellTools`` are not tested with actual ``Device<Cuda,CudaSpace>``.